tortoise/verifying_tortoise.go

```diff
@@ -84,13 +67,11 @@ func (t *turtle) init(genesisLayer *types.Layer) {
 	t.BlockOpinionsByLayer[genesisLayer.Index()] = make(map[types.BlockID]Opinion)
 	for _, blk := range genesisLayer.Blocks() {
 		id := blk.ID()
-		t.BlockOpinionsByLayer[genesisLayer.Index()][blk.ID()] = Opinion{
-			BlocksOpinion: make(map[types.BlockID]vec),
-		}
-		t.GoodBlocksIndex[id] = struct{}{}
+		t.BlockOpinionsByLayer[genesisLayer.Index()][id] = Opinion{}
+		t.GoodBlocksIndex[id] = false
```

Comment:
this should be `true` shouldn't it?

Reply:
true is for flushed

Reply:
I don't understand. this line (and the other, similar one) very clearly looks like it's marking the block with id `id` as good. am I missing something?

Reply:
so if block is present in this map it means that the block is good. if the value is set to true it means that it was flushed to the disk, and doesn't need to be flushed again

Reply:
okay. that's counterintuitive from the design. please document that clearly :) thanks!